src/main/java/net/glowstone/io/entity/LivingEntityStore.java

```diff
@@ -148,6 +130,68 @@ public void load(T entity, CompoundTag compound) {
         }
     }
 
+    private void loadEquipment(T entity, EntityEquipment equip, CompoundTag compound) {
+        // Deprecated since 15w31a, left here for compatibilty for now
+        if (compound.isList("Equipment", TagType.COMPOUND)) {
+            List<CompoundTag> list = compound.getCompoundList("Equipment");
+            if (list.size() == 5) {
```

Comment:
The only situation I could see this happening is when using data-tags in commands (/summon, for example): it would be possible for a command to not have all the items in the list, and the Entity Store would have to handle that.